src/assets/styles/components.scss

```diff
@@ -20,11 +20,11 @@
 }
 
 .entity-status.is-degraded {
-  color: var(--yellow-300);
+  color: #ff8040;
```

Comment:
just a thought, `--yellow-400` won't work for this?

Reply:
not really :D 
![image](https://user-images.githubusercontent.com/12110261/107750955-4a389c00-6d4f-11eb-9131-03ef38f5eb13.png)

Reply:
I referred kongponents for the color code - https://kongponents.konghq.com/style-guide/colors.html#colors for `yellow-400`.

Reply:
Oh I see, that's weird, I think `kuma-gui` depends on the previous version of kongponents, there are only 4 yellow colors like on the screenshot